.github/workflows/integration.yml

```diff
@@ -59,8 +59,3 @@ jobs:
           FOLDER=/builds/test_gha_dxcompiler_$(date +%s)_$RANDOM
           dx mkdir $FOLDER
           python3 scripts/run_tests.py --test M --folder $FOLDER --project $PROJECT
-      - name: Upload artifact
```

Comment:
@commandlinegirl  previously used this for running some tests manually, but it isn't needed anymore. @jdidion